app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -103,6 +114,22 @@ fun RecentTabs(
                         )
                     }
                 }
+                is RecentTab.SyncedTab -> {
+                    if (FeatureFlags.taskContinuityFeature) {
+                        RecentSyncedTabItem(
+                            tab,
+                            onRecentSyncedTabClick,
+                            onSyncedTabSeeAllClick,
+                        )
+                    }
+                }
+            }
+            if (FeatureFlags.taskContinuityFeature) {
+                RecentSyncedTabItem(
+                    null,
+                    onRecentSyncedTabClick,
+                    onSyncedTabSeeAllClick,
+                )
```

Comment:
Since this is inside then `when` block, I don't think it was intended for this to remain here.